daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -404,7 +401,7 @@ class Engine(val config: EngineConfig = Engine.StableConfig) {
                 { granted: Boolean =>
                   if (granted) {
                     callback()
-                    loopOuter(cache)
```

Comment:
This was previously:
interpretLoop(machine, time)

Any reason for changing this back differently?

Reply:
a07f8f1d2e1d04b416733fcf322b80928ccdfb57 reverts back the previous version